ai-ml/jark-stack/terraform/README.md

```diff
@@ -1,6 +1,79 @@
 # JupyterHub, Argo, Ray, Kubernetes
 
-Docs coming soon...
+# Prerequisites
```

Comment:
Lets remove this doc as well. We have an open [issue](https://github.com/awslabs/data-on-eks/issues/445) to replace the existing Ray on EKS blueprint with JARK stack along with the  documentation in the website.